package mad_manager.messages;
option optimize_for = LITE_RUNTIME;


message ipcp_config{  //Contains the information to setup a new IPCP Process
	required string process_name = 1;  			//the name of the process
	required uint32 process_instance = 2;  		//the instance of the process
	required string process_type = 3;  			//the type of the process
	required string dif_to_register = 4;  		//the DIF to register the IPCP
	required string dif_to_assign = 5;  		//the DIF to assign the IPCP
}